sdk/storage/Azure.Storage.Common/src/Errors.cs

```diff
@@ -40,5 +53,37 @@ internal class Errors
 
         public static ArgumentException InvalidResourceType(char s) =>
             new ArgumentException($"Invalid resource type: '{s}'");
+
+        public static ArgumentException InvalidDateTimeUTC(string dateTime) =>
```

Comment:
Nit - in .NET we'd format this as `InvalidDateTimeUtc` since acronyms with more than two letters only get the first letter capitalized.

Reply:
Updated. Changed to InvalidDateTimeUtc